### ISSUE TITLE
No tags shown in releases page

### ISSUE THREAD

**Issue by mueller-ma** [slapperwan/gh4a#1077]
On `github.com/<orga>/<repo>/releases` there are releases and tags without an associated release, but OctoDroid only shows releases there.

Examples:

- Repo with releases only: https://github.com/slapperwan/gh4a/releases ✔️ 
- Repo with tags only: https://github.com/mueller-ma/Coffee/releases ❌ 
  - OctoDroid shows nothing there, GitHub all tags
- Repo with releases and tags, that aren't associated to releases: https://github.com/openhab/openhab-android/releases :x:

**Comment by maniac103** [comment]
I think the only meaningful way of implementing this is doing the release list in two tabs (releases/tags), where the entries in the latter jump to the code browser with that tag selected. Would you agree?

**Comment by mueller-ma** [comment]
Agreed. Currently OctoDroid doesn't handle `github.com/<orga>/<repo>/tags` at all, right?

**Comment by maniac103** [comment]
No, it doesn't. I could add that when adding that new tab.
